### PR TITLE
FAQ page

### PR DESCRIPTION
**Pull request by TheFlash98**

### CODE REVIEW

File: app/src/main/res/layout/faqpage.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of letrashet sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Adlus Pagemaker including versions of Lorem Ipsum."
+        android:textColor="#FFFFFF"
+        android:textSize="20dp"
+        android:paddingTop="10dp"
+        android:paddingRight="15dp"
+        android:paddingLeft="15dp"
+        />
```
Comment (arunabh98) [inline app/src/main/res/layout/faqpage.xml]: Shift `/>` to the line above.

**Review by arunabh98 — CHANGES_REQUESTED** [review]

**Comment by arunabh98** [comment]
@TheFlash98 this is great! Just a few things to take care of...
 Could you rename `faqpage.xml` to `activity_faq.xml`. Also could you create a Activty file corresponding to this layout - `FaqActivity`.

File: app/src/main/res/layout/faqpage.xml (hunk 1 above)
Comment (unstablebrainiac) [reply app/src/main/res/layout/faqpage.xml]: We'll reformat the code occasionally, this shouldn't be a problem.

**Comment by unstablebrainiac** [comment]
Instead of hardcoding strings, write them in the `strings.xml` file and use `@string` in `android:text` attribute.
Also consider using standard spacing and padding sizes following [material design guidelines](https://material.google.com/layout/metrics-keylines.html)

**Comment by nihal111** [comment]
Use Ctrl + Shift + Alt + L for code cleanup

File: app/src/main/res/layout/faqpage.xml (hunk 1 above)
Comment (arunabh98) [reply app/src/main/res/layout/faqpage.xml]: Yes you're right. But waise bhi he has to make other changes so he might as well do this.

**Comment by arunabh98** [comment]
I tested it and I think this looks good!

**Review by arunabh98 — APPROVED** [review]

**Comment by unstablebrainiac** [comment]
Can you replace the activity with a fragment so that it can be linked with the navigation drawer?

**Pull request merged**